CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file. If a contri
 
 ## Unreleased
 
+- Use react-is.isValidElementType to check whether elements passed to constructWithOptions are valid (see [#1658]https://github.com/styled-components/styled-components/pull/1658)
```

Comment:
Small markdown typo in the link here. Can we also describe what this achieves and not what it changes please? In this case it’d be great to say that the styled constructor has been improved to accept ref forwarding components :)